make/utils_exposed.cmake

```diff
@@ -80,14 +80,20 @@ function (DynamoRIO_add_rel_rpaths target)
   if (UNIX AND NOT ANDROID) # No DT_RPATH support on Android
     # Turn off the default CMake rpath setting and add our own LINK_FLAGS.
     set_target_properties(${target} PROPERTIES SKIP_BUILD_RPATH ON)
+
     foreach (lib ${ARGN})
       # Compute the relative path between the directory of the target and the
       # library it is linked against.
-      get_target_property(tgt_path ${target} LOCATION)
-      get_target_property(lib_path ${lib} LOCATION)
-      _DR_dirname(tgt_path "${tgt_path}")
-      _DR_dirname(lib_path "${lib_path}")
-      file(RELATIVE_PATH relpath "${tgt_path}" "${lib_path}")
+
+      # XXX: is this here needed for win at all? It shouldn't, no rpath
```

Comment:
Remove this line (and the blank line): the answer is no and Windows is already excluded by line 80 above.